### DISCUSSION TITLE
Is it possible to generate visualization of scalp and electrodes?

### DISCUSSION THREAD

**Discussion by AnantShinde** [simnibs/simnibs#146] [9. Results and analysis]
I want to create an image that shows electrode position and scalp but without an electric field.
Like realistic head/scalp and overlay electrodes. Could you help me with this? 
I have ran simulation using Ernie bu placing two electrodes and tried many visualization options in Gmsh.

**Comment by oulap** [comment]
Hi,

You can tick off all the additional views (E, magnE..), which will leave you with no visualization initially. Then go to Tools->Visibility and select the structures you want to visualize, e.g., scalp and electrodes. You can also select all structures and click apply. Next go to Tools -> Options and click on "Mesh". Then select the "Visibility" tab and click on "2D element faces". Now you should see the electrodes and scalp surface. You can also visualize the volume elements by clicking on "3D element faces".

You can also check the small gmsh visualization tutorial we have on the website:

https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html
